.betterer.ts

```diff
@@ -86,6 +86,18 @@ function countEslintErrors() {
       '@emotion/syntax-preference': [2, 'object'],
       '@typescript-eslint/no-explicit-any': 'error',
       '@grafana/no-aria-label-selectors': 'error',
+      'no-restricted-imports': [
+        'error',
+        {
+          patterns: [
+            {
+              group: ['@grafana/ui/*'],
```

Comment:
seems like way too few matches added to the betterer.results 

wonder if the pattern should instead be (note the absence of the trailing slash)
```suggestion
              group: ['@grafana/ui*'],
```